data/avgi/avgi outfits.txt

```diff
@@ -0,0 +1,1488 @@
+
+
+outfit "Concentrating Solar Array"
+	category "Power"
+	licenses
+		"Avgi Civilian"
+	cost 200000
+	thumbnail "outfit/unknown"
+	"mass" 3
+	"outfit space" -18
+	"solar collection" 2.4
+	"solar heat" 1.2
+	description "This large set of concentrating solar cells efficiently collects sunlight, turning it into usable energy. Thin Fresnel lenses concentrate sunlight onto a small number of highly efficient solar cells, reducing weight and improving efficiency."
+	description "While much more efficient than a single cell alone, the Solar Array also produces more than a proportionate amount of heat."
+
+
+outfit "Photoelectric Scales"
+	plural "Photoelectric Scales"
+	category "Power"
+	licenses
+		"Avgi Civilian"
+	cost 200000
+	thumbnail "outfit/photoelectric scales"
+	"mass" 24
+	"anchor point" -1
+	"solar collection" 6
+	"solar heat" 1.8
+	description "These concentrating solar panels can be mounted conformally to a ship's hull, efficiently collecting sunlight and turning it into usable energy."
+	description "Photovoltaic Scales take advantage of special external hardpoints built into most Avgi ships, allowing them to be mounted without taking up any outfit space. However, they also tend to increase the amount of heat absorbed from the local star. However, they also tend to increase the amount of heat absorbed from the local star."
+
+
+outfit "Lantern Fission Core"
+	category "Power"
+	licenses
+		"Avgi Civilian"
+	cost 1200000
+	thumbnail "outfit/lantern fission core"
+	"mass" 24
+	"outfit space" -24
+	"energy generation" 3.6
+	"heat generation" 5.4
+	description "The Lantern is an extremely compact dusty plasma fission reactor designed to fit even on large fighters or interceptors. By suspending uranium dust in a hydrogen plasma, which is in turn contained by a magnetic field, the reactor can achieve core temperatures high enough to make it efficient for its size."
+
+
+outfit "Luminous Fusion Core"
+	category "Power"
+	licenses
+		"Avgi Civilian"
+	cost 20460000
+	thumbnail "outfit/fusion core small"
+	"mass" 36
+	"outfit space" -36
+	"energy generation" 12
+	"heat generation" 30
+	"fuel consumption" 0.0033
+	description "Balance TBD"
+	
+
+outfit "Brilliant Fusion Core"
+	category "Power"
+	licenses
+		"Avgi Military"
+	cost 20460000
+	thumbnail "outfit/unknown"
+	"mass" 72
+	"outfit space" -72
+	"energy generation" 21
+	"heat generation" 54
+	"fuel consumption" 0.0042
+	description "Balance TBD"
+
+
+outfit "Radiant Fusion Core"
+	category "Power"
+	licenses
+		"Avgi Military"
+	cost 20460000
+	thumbnail "outfit/unknown"
+	"mass" 120
+	"outfit space" -120
+	"energy generation" 36
+	"heat generation" 96
+	"fuel consumption" 0.0066
+	description "Balance TBD"
+
+
+outfit "Stellar Fusion Core"
+	category "Power"
+	licenses
+		"Avgi Military"
+	cost 20460000
+	thumbnail "outfit/unknown"
+	"mass" 144
+	"outfit space" -144
+	"energy generation" 54
+	"heat generation" 156
+	"fuel consumption" 0.009
```

Comment:
Even for this, you can fuel three of these with the fuel generation of one electrostatic ramscoop, and the electrostatic ramscoop is weaker in most circumstances than existing ramscoops. I do think these reactors could be made to work (though, probably should be using `fuel energy` and `fuel heat`), but I think you need to be thinking bigger with the fuel consumption.

Reply:
something something, did you know fuel energy and fuel heat to cause the energy and heat to stop when you run out of fuel?